test-clients/src/main/java/com/hedera/services/yahcli/config/domain/NetConfig.java

```diff
@@ -28,6 +28,7 @@ public class NetConfig {
 
     private String defaultPayer;
     private Integer defaultNodeAccount = TRADITIONAL_DEFAULT_NODE_ACCOUNT;
+    private List<Long> allowList;
```

Comment:
This needs a clearer name. 'allowedReceiverAccountIds'?

Reply:
thank you will fix it

Reply:
done